apps/web/components/booking/CancelBooking.tsx

```diff
@@ -34,7 +33,6 @@ export default function CancelBooking(props: Props) {
   const [loading, setLoading] = useState(false);
   const telemetry = useTelemetry();
   const [error, setError] = useState<string | null>(booking ? null : t("booking_already_cancelled"));
-  useTheme(props.theme);
```

Comment:
No need to use `useTheme` here. It's used by booking/[uid] and that has `useTheme` call.